### PR TITLE
[Ready] brittle plasma bones

### PR DESCRIPTION
**Pull request by lilacrhododendron**
greetings, i've come with yet another one line quick fix balance pr

lots of people said that plasmamen are overpowered now, and some people missed the reputation that they used to have. this change is intended to deter people from playing them for "powergaming" purposes by making them tangibly more fragile while avoiding the annoying speed slowdown.

there is probably a more creative drawback idea out there, if you hate this pr and you've got one i highly suggest you go hog wild with it

:cl: the epic purple skeleton in the sky
balance: Plasmamen now take 1.5x brute damage.
/:cl:

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
If you manage to give them a deathrattle sound that sounds like a cartoony xylophone skeleton collapsing into a pile that would be funny.

**Comment by subject217** [comment]
that is a hilarious suggestion, sounds are not my forte though

**Comment by nicbn** [comment]
They already require a room filled with plasma to be cloned, that's enough

I don't like the tweak values balance, more creative balances that use different mechanics are better.

For example: adding a chance (10% or something like that) to set them on fire by damaging them, limiting their vision (by like 1 or 2 tiles) if they use the helmet, making them explode on fire if reached by an explosion, making their bodies explode if exposed to high temperatures, making need their own snowflake medication etc.

**Comment by GuyonBroadway** [comment]
Ya don't need a plasma room. Just give em some salbutomol and cryox and bung them under a cold shower.

**Comment by Moja10** [comment]
Serious question that I am wondering: are people really saying plasmamen are overpowered?

**Comment by necromanceranne** [comment]
So plasmamen take half as much damage from everything? Okay....

They already had the burn mod and burn damage is some of the widest available in ranged weaponry for the crew, and welders are very good weapons too. Their helmets are also flash immune because they cant use welder masks. Not sure why they need to die to toolboxes faster specifically when a few of their advantages could be fairly minor changes elsewhere (like making their masks flashproofing be something they need to bring down and it restricts vision, much like an actual welder mask).

I really don't believe the hype. Plasmamen are used a lot for engineering and atmos memes because they're rad immune, not because the race is inherently more powerful. There are a few places they do have roundstart advantages. Altering those would be a better priority and a better fix.

**Comment by Tlaltecuhtli** [comment]
no tags and misleading title

**Comment by imsxz** [comment]
as someone whos been abusing new plasmamen to the fullest, this PR is good but im not sure it's the direction I'd have chosen. some nerfs I'd have made instead would be removing the inbuilt flash protection and ear protection on their helmet.

**Comment by subject217** [comment]
>It appears to be that this also removes their calcium healing and always clean traits, is that intentional?

Nope, it's cause I made this commit a couple of days ago and something for then got merged, I'll fix it in just a bit.

>no tags and misleading title

I like my title. No tags is MSO's fault for changing PR formatting, it broke the auto-tagger. It's in the changelog, any maintainer can obviously add the tag.

Furthermore, whether or not le epic reddit squad shows up to throw insults at me is irrelevant. It has zero bearing on whether it gets merged or not. The change is straightforward and extremely simple.

@imsxz
Since this is a trite change I decided it would be better to avoid touching their *flavor*. It is a little annoying with regards to flash protection though, Plasmamen can't wear welding helmets. If you made welding goggles constructible it sounds like a good idea to me, you should make a PR for it.

File: code/modules/mob/living/carbon/human/species_types/plasmamen.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	sexes = 0
 	meat = /obj/item/stack/sheet/mineral/plasma
 	species_traits = list(NOBLOOD,NOTRANSSTING)
-	inherent_traits = list(TRAIT_RESISTCOLD,TRAIT_RADIMMUNE,TRAIT_NOHUNGER,TRAIT_CALCIUM_HEALER,TRAIT_ALWAYS_CLEAN)
+	inherent_traits = list(TRAIT_RESISTCOLD,TRAIT_RADIMMUNE,TRAIT_NOHUNGER)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/plasmamen.dm]: Why did you remove these?

File: code/modules/mob/living/carbon/human/species_types/plasmamen.dm [hunk 2]
```diff
@@ -15,7 +15,9 @@
 	blacklisted = 1 //See above
 	burnmod = 1.5
 	heatmod = 1.5
+	brutemod = 1.5
 	breathid = "tox"
+	speedmod = 1
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/plasmamen.dm]: And add this?

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/plasmamen.dm (hunk 2 above)
Comment (subject217) [reply code/modules/mob/living/carbon/human/species_types/plasmamen.dm]: both are accidents because my branch is old and I was under the influence

**Comment by zxaber** [comment]
Plasma men can wear any flavor of gas masks, though, right? Because the welding gas mask totally _is_ printable from the science (and probably engineering) protolathe, and almost always fairly early into the round since their tech node is on the path to mining tech.

**Comment by Qustinnus** [comment]
@subject217 I dislike the use of brute mod for plasmamen because Ethereal are already weak to brute, why dont we instead focus on their weakness to fire so that you actually have to consider different weapons for fighting them instead of just "brute works on anything"

**Comment by PKPenguin321** [comment]
I like that idea qustinnus. Plasma is really volatile and they're made of the stuff. Maybe burn damage attacks of any kind could have a chance to ignite, scaling with their damage?

**Comment by subject217** [comment]
I thought about that. If you want to make your own PR for it, it's not a bad idea thematically, but I think it's more important from a balance perspective that they're vulnerable to the most common form of damage, mostly to explicitly discourage people from playing them in order to powergame.

**Comment by Qustinnus** [comment]
I feel like it indirectly pollutes the ethereal weakness, I'd prefer a balance that doesn't make the race to similar to ethereals in terms of what to counter them with

**Comment by Tlaltecuhtli** [comment]
welding goggles exist

**Comment by Ziiro** [comment]
If you really want to fuck powergamers that go for it, I'd suggest using `stunmod = 1.1`. It *really* fucks people up.
